examples/typescript-reactflux/externs/react.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Created by bparadie on 3/1/15.
+ */
+
+/**
+ * @type {*}
+ */
+var __REACT_DEVTOOLS_GLOBAL_HOOK__;
+
+/**
+ * @type {*}
+ */
+var Symbol;
+
+/**
+ * @type {*}
+ */
+var MSApp;
```

Comment:
@arthurvr Same thing: those extern files are for the closure compiler. Otherwise you can't compile with ADVANCED_OPTIMIZATTION. I couldn't find any extern definitions for react (happens a lot), there are no npm modules.